servers_v8.json

```diff
@@ -151,5 +151,9 @@
   {
     "name": "Aetherial",
     "address": ["188.127.241.22:25637", "188.127.241.22:25711", "188.127.241.22:25662", "188.127.241.22:25733"]
+  },
+  {
+    "name": "Rosemoncorp Servers",
+    "address": ["194.164.245.234:6567", "194.164.245.234:6568"]
```

Comment:
The server "194.164.245.234:6568" is not responding to pings, joining it results in being taken to the same server as :6567 (a sandbox server)

<img width="1390" height="259" alt="image" src="https://github.com/user-attachments/assets/f52a2f49-b9cf-42c1-894f-5fa025dff1e0" />

Are you trying to add 1 server or 2 servers?

If you are trying to add 2 servers, there is something wrong with your second server, it just goes to the first server.

If you are trying to add 1 server, please remove the second entry from this list, it is useless and may cause your server to get displayed twice.